### ISSUE TITLE
[linux] Selecting audio sink as a source

### ISSUE THREAD

**Issue by Kuro-Maii** [Neos-Metaverse/NeosPublic#2665]
### Describe the bug?

When opening the settings and going to select audio input you'll find a list of system default and a couple of other audio devices, some of these are actually output devices and not input.
selecting an output (sink) has a change of making neos freeze for a second.
if you have a "simultaneous output device", one that sends the audio to more then one audio sink, and select that as an input for neos it will hard freez the game and needs a `kill -9` 

### Relevant issues

none that I could find

### To Reproduce

while in VR on linux:
- open dash
- goto settigns
- open "select input devices"
- select any of the outputs
- - if an actual output neos will freeze for a short moment
- - if an simultaneous output noes will lockup
- occasionally it will just lockup on selecting an actual audio source when having "successfully" selected an output as input

### Expected behavior

exclusively seeing inputs (audio sources) and no outputs (audio sinks)

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

Beta 2021.7.11.1002

### What Platforms does this occur on?

Linux

### Did this work before?

No

### Additional context

if there are no simultaneous outputs available that can be created with PulseAudio Preferences

### Reporters

neos: Kuro Maii
discord: Kuro Maii#6123

**Comment by shadowpanther** [comment]
The output devices (sinks) are meant to capture the audio playing on your computer and streaming that into Neos. Windows version has those marked as "(loopback)". The intended way of using those is for the audio streaming panel, which is separate from your main voice stream.

**Comment by Frooxius** [comment]
Are you running the native Linux build or running the Windows version through Proton?

The reason is that we don't have custom audio input driver on Linux, so I'm a bit confused how could those devices be appearing there in the first place.

**Comment by Kuro-Maii** [comment]
@shadowpanther there is a separate "loopback" listed 
@Frooxius I am using the native linux build

the only device that works is the default input
of the other devices listed only the "valve...", "USB Headset Mono" and "HP 720p..." are known inputs.
the 2 USB devices listed is the headset I use so one of them is an output
steamVR messes up the audio in/out on the Index. The index's mic actually never works....
![image](https://user-images.githubusercontent.com/4559380/126080474-d177be9c-a0e2-4909-99e6-7cfcc5045ce0.png)
appologies for the poormans screenshot, making a quick desktop screengrab was the only way I could quickly think of :sweat_smile:

**Comment by shadowpanther** [comment]
Yes, that `Neos Loopback` is an internal one, I think (sound not leaving the client at all, just gets rerouted) and is also present on Windows.

**Comment by Kuro-Maii** [comment]
yhea I understand the function of the loopback and that isn't a problem. 
it is the actual output devices like all of the "monitor of...." and the one with the massive name at the top (2nd item) which is the "simultaneous output device" those are being problematic and shouldn't even be there ^,,^

I was kinda hoping that neos would somehow be able to pickup the mic from the index (several 3rd parties have made all the "broken" parts work under linux and vale isn't implementing those fixes.... >,,> _looks at valve_ ).
I had no other reason to be in that menu as I am perfectly understandable with that old USB mic on the default input, and that is the reason I discovered this and it is in my nature to poke around if I see weird stuff (i.e. the outputs listed under inputs)
